tests/emukit/quadrature/test_quadrature_acquisitions.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+
+import numpy as np
+import pytest
+import GPy
+from math import isclose
+
+from emukit.model_wrappers.gpy_quadrature_wrappers import QuadratureRBF, RBFGPy, BaseGaussianProcessGPy
+from emukit.quadrature.methods import VanillaBayesianQuadrature
+from emukit.quadrature.acquisitions import MutualInformation, IntegralVarianceReduction
+
+
+REL_TOL = 1e-5
+ABS_TOL = 1e-4
+
+
+@pytest.fixture
+def model():
+    rng = np.random.RandomState(42)
+    x_init = rng.rand(5, 2)
+    y_init = rng.rand(5, 1)
+
+    gpy_kernel = GPy.kern.RBF(input_dim=x_init.shape[1])
```

Comment:
In the tests, an alternative to building a model is to use Mock. It is Ok in this case as the model is pretty though.